extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1110,14 +1036,16 @@ export class AmpLightboxGallery extends AMP.BaseElement {
           'keydown', this.boundOnKeyDown_);
 
       this.carousel_.addEventListener(
-          'slideChange', event => this.slideChangeHandler_(event)
-      );
+          'slideChange', this.boundSlideChangeHandler_);
 
       this.setupGestures_();
       this.setupEventListeners_();
 
       return this.carousel_.signals().whenSignal(CommonSignals.LOAD_END);
-    }).then(() => this.openLightboxForElement_(element))
+    })
+        .then(() => {
+          return this.openLightboxForElement_(element, expandDescription);
```

Comment:
Any reason for expanding the short return syntax? I find that the short syntax can diminish the temptation to make the callback more complex and instead encapsulate behavior in the callee.

Either way, just a personal preference thing.